installer/install.iss

```diff
@@ -2166,14 +2166,14 @@ begin
     RdbGitCredentialManager[GCM_Core]:=CreateRadioButton(GitCredentialManagerPage,'Git Credential Manager Core','<RED>(NEW!)</RED> Use the new, <A HREF=https://github.com/microsoft/Git-Credential-Manager-Core>cross-platform version of the Git Credential Manager</A>.'+#13+'See more information about the future of Git Credential Manager <A HREF=https://github.com/microsoft/Git-Credential-Manager-Core/blob/master/docs/faq.md#about-the-project>here</A>.',TabOrder,Top,Left);
 
     // Git Credential Manager for Windows
-    RdbGitCredentialManager[GCM_Classic]:=CreateRadioButton(GitCredentialManagerPage,'Git Credential Manager','(DEPRECATED) The <A HREF=https://github.com/Microsoft/Git-Credential-Manager-for-Windows>Git Credential Manager for Windows</A> handles credentials e.g.'+#13+'for Azure DevOps and GitHub (requires .NET framework v4.5.1 or later).',TabOrder,Top,Left);
+    RdbGitCredentialManager[GCM_Classic]:=CreateRadioButton(GitCredentialManagerPage,'Git Credential Manager','(DEPRECATED) The <A HREF=https://github.com/Microsoft/Git-Credential-Manager-for-Windows>Git Credential Manager for Windows</A> handles credentials e.g.'+#13+'for Azure DevOps and GitHub (requires .NET framework v4.6.1 or later).',TabOrder,Top,Left);
```

Comment:
This seems untrue. This text is about the deprecated [Git Credential Manager for Windows](https://github.com/microsoft/Git-Credential-Manager-for-Windows), not the newer [Git Credential Manager Core](https://github.com/microsoft/Git-Credential-Manager-Core) you mentioned in the commit message. This GCM Classic does seem to [only require .NET Framework 4.5.1](https://github.com/microsoft/Git-Credential-Manager-for-Windows/blob/master/Installer/Setup.iss#L211) as stated in the original Text.

The GCM Core text that should probably mention this reqjuirement is 3 lines above.